tests/requirements.txt

```diff
@@ -0,0 +1,6 @@
+docker==7.1.0
+GPUtil==1.4.0
+jinja2==3.1.4
+pytest==8.3.2
+nvidia-ml-py==12.560.30
+transformers==4.44.2
```

Comment:
for what do we need transformers here?

Reply:
We used that for the prompt formatting, but we could remove it as we don't actually care about that I guess!